Error appears when trying to delete a prescription linked to a treatment

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
Error appears when trying to delete a prescription linked to a treatment

### Steps to Reproduce

1. Login as one of the roles: NATIONAL_CLINICIAN,  CASE_SUPERVISOR, ADMIN
2. Edit/create a Case
3. Go to Therapy tab
4. Create a Prescription
5. Create Treatment from the Prescription created above
6. Open in edit mode the Treatment created 
7. Select Open prescription
8. Try to delete the prescription

### Actual Behavior
The prescription remains in edit mode without any effect and an error appear in the server logs

### Expected Behavior
When trying to delete a prescription linked to the treatment, a confirmation pop-up should appear asking if user wants to deleted also the linked treatment. If the answer is no, then the treatment will remain in the system as standalone. If the answer is yes, then the treatment will be deleted together with the prescription

### Screenshots
<!-- Optional -->

### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device:
* SORMAS version: 1.71.0
* Android version/Browser:
* Server URL:
* User Role: NATIONAL_CLINICIAN,  CASE_SUPERVISOR

### Additional Information
<!-- Optional -->
[DeleteError.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/8572838/DeleteError.txt)
```
Caused by: org.postgresql.util.PSQLException: ERROR: update or delete on table "prescription" violates foreign key constraint "fk_treatment_prescription_id" on table "treatment"
  Detail: Key (id)=(2298) is still referenced from table "treatment".
```

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/therapy/TherapyController.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/therapy/TherapyController.java
@@ -138,7 +138,7 @@ public class TherapyController {
 				new Label(I18nProperties.getString(Strings.confirmationDeletePrescriptionWithTreatment)),
 				I18nProperties.getCaption(Captions.prescriptionAlone),
 				I18nProperties.getCaption(Captions.prescriptionWithTreatment),
-				null,
+				650,
 				resultConsumer,
 				true);
 	}

